### PR TITLE
Added Pre-Work challenge.md

### PR DESCRIPTION
**Pull request by NikhilSharma03**
## What does this change?

Module:HTML/CSS
Week(s):Pre-Work

## Description
Added Pre-Work to the HTML/CSS module so that students can refresh their knowledge by completing a simple HTML/CSS exercise which should take around 3 to 4 hours. It will also contribute to the course work hours.

## Issue

#247

### CODE REVIEW

**Review by ChrisOwen101 — CHANGES_REQUESTED** [review]
Hi @NikhilSharma03 

This is a really good start but I think it could do with some more fleshing out. Maybe take a look at our React Challenges for some examples of what you might include

https://github.com/CodeYourFuture/cyf-react-challenges/tree/master/challenge-baby-name-picker

Ideally, the challenge should have an example solution and more strict guidelines on what we need of the students. 

Also, you'll want to add this page to the Sidebar of the HTML module. You can find out how to do this here

https://docusaurus.io/docs/2.0.0-alpha.75/sidebar

Give me a nudge when you've made you're changes!

File: docs/html-css/pre-work/challenge.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+-About Page
+Information about the user such as interest, hobbies, technical experience
+
+-Skill Page
+List all the languages and frameworks the user is familiar with
+
+-Contact Page
+List all the social media links such as GitHub, LinkedIn, and so on.
+
+**The requirement is that the website should be responsive and have a decent theme with a regular color combination**
```
Comment (ChrisOwen101) [inline docs/html-css/pre-work/challenge.md:17]: Students won't have learnt about Responsive Design yet since this is before the HTML module

Comment (ChrisOwen101) [inline docs/html-css/pre-work/challenge.md:17]: Additionally, what does the word "regular" mean here?

**Review by ChrisOwen101 — CHANGES_REQUESTED** [review]

File: docs/html-css/pre-work/challenge.md (hunk 1 above)
Comment (NikhilSharma03) [reply docs/html-css/pre-work/challenge.md:17]: Sorry, it was a typo. I will update it.

Comment (NikhilSharma03) [reply docs/html-css/pre-work/challenge.md:17]: Ok, I will update it.

**Comment by ChrisOwen101** [comment]
Closing this PR as inactive. @NikhilSharma03 you're welcome to re-open when you've had some time to work on this 😊

**Pull request closed without merging**